specification/security/resource-manager/Microsoft.Security/preview/2019-01-01-preview/networkData.json

```diff
@@ -0,0 +1,400 @@
+      }
+    },
+    "NetworkDataConnectableResource": {
+      "type": "object",
+      "description": "Describes the allowed inbound and outbound traffic of an Azure resource",
+      "properties": {
+        "connectedResourceId": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The connectable resource id"
+        },
+        "connectionType": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The connection type"
+        },
+        "allowedConnections": {
+          "readOnly": true,
+          "type": "object",
+          "description": "The allowed connections details",
+          "properties": {
+            "calculatedDateTime": {
+              "type": "string",
+              "format": "date-time",
+              "readOnly": true,
+              "description": "The UTC time on which the allowed connections was calculated"
+            },
+            "inboundConnectedResources": {
+              "readOnly": true,
+              "type": "object",
+              "description": "The allowed connections inbound details",
+              "properties": {
+                "tcpPortsSummary": {
+                  "readOnly": true,
+                  "type": "integer",
+                  "description": "The allowed connections inbound tcp ports summary"
+                },
+                "udpPortsSummary": {
+                  "readOnly": true,
+                  "type": "integer",
+                  "description": "The allowed connections inbound udp ports summary"
+                },
+                "tcpPorts": {
+                  "readOnly": true,
+                  "type": "string",
+                  "description": "The allowed connections inbound tcp ports"
+                },
+                "udpPorts": {
+                  "readOnly": true,
+                  "type": "string",
+                  "description": "The allowed connections inbound udp ports"
+                }
+              }
+            },
+            "outboundConnectedResources": {
+              "readOnly": true,
+              "type": "object",
+              "description": "The allowed connections outbound details",
+              "properties": {
+                "tcpPortsSummary": {
+                  "readOnly": true,
+                  "type": "integer",
+                  "description": "The allowed connections outbound tcp ports summary"
+                },
+                "udpPortsSummary": {
+                  "readOnly": true,
+                  "type": "integer",
+                  "description": "The allowed connections outbound udp ports summary"
+                },
+                "tcpPorts": {
+                  "readOnly": true,
+                  "type": "string",
+                  "description": "The allowed connections outbound tcp ports"
+                },
+                "udpPorts": {
+                  "readOnly": true,
+                  "type": "string",
```

Comment:
did you consider making tcpPorts and udpPorts as string array?

Reply:
yes.
There are optional values like "*" (any) so we preferred that those fields will be string